src/sfa.Roatp.Register.ApiIntegrationTests/Steps/Stepbindings.cs

```diff
@@ -0,0 +1,142 @@
+﻿using BoDi;
+using System.Collections.Generic;
+using System.Linq;
+using TechTalk.SpecFlow;
+using TechTalk.SpecFlow.Assist;
+using System;
+using System.Globalization;
+using NUnit.Framework;
+using Sfa.Roatp.Register.Web.Controllers;
+using sfa.Roatp.Register.ApiIntegrationTests.Infrastructure;
+using SFA.Roatp.Api.Types;
+using System.Web.Http;
+using Esfa.Roatp.ApplicationServices.Models.Elastic;
+
+namespace sfa.Roatp.Register.ApiIntegrationTests.StepBindings
+{
+    [Binding]
+    public class Stepbindings
+    {
+        private readonly IObjectContainer _objectContainer;
+
+        public Stepbindings(IObjectContainer objectContainer)
+        {
+            _objectContainer = objectContainer;
+        }
+
+        #region Given
+
+        [Given(@"the following roatp providers are available")]
+        public void GivenTheFollowingRoatpProvidersAreAvailable(Table table)
+        {
+            var stubrepo = _objectContainer.Resolve<StubProviderRepository>("StubRepo");
```

Comment:
it's not needed now but in the near future we'll probably refactor the mapping into a class in the container.

could you use the 

```c#
var container = IoC.Initialize();
container.Configure(x => x.For<IGetProviders>().Use<StubProviderRepository>());
```
If that registration doesn't work you may need to do a clear on the previous registration

```c#
For<IWidget>().ClearAll().Use<ImportantClientWidget>();
```